readme.md

```diff
@@ -64,21 +64,15 @@ To generate CPPCheck and CPPLint code analysis:
 
 ```bash
 # from the BashScripts directory
```

Comment:
These are all currently written to be run from the BashScripts directory; running from the root directory will cause the build to crash. Was requested to move them into separate directory by Daniel.

Reply:
That's my bad; I totally missed the comment in the code. I'll revert my changes.